packages/calcite-components/src/utils/label.ts

```diff
@@ -166,7 +166,13 @@ function sortByDOMOrder(a: LabelableComponent, b: LabelableComponent): number {
  * @param component
  */
 export function getLabelText(component: LabelableComponent): string {
-  return component.label || component.labelEl?.textContent?.trim() || "";
+  return (
+    component.label ||
+    component.labelEl?.textContent?.trim() ||
+    component.el?.getAttribute("label") ||
```

Comment:
I'm not sure we need to add these.

```
component.el?.getAttribute("label") ||
component.el?.getAttribute("aria-label") ||
```

I would suspect users should be setting the `label` property on the host and not the `aria-` attributes because that would lead to multiple identical `aria-` attributes on the host and within the host.

If we do introduce this, we would need to update the label tests to include this logic

Reply:
Agreed, thanks for the feedback on that change.  Adding the label property to input-time-picker addresses this by itself, but I was curious to hear what others thought about this change, which catches components that don't have a `@property() label` defined, but we should probably audit all components to make sure the ones that should have labels have the `@property() label` class member just so the `getLabelText` can properly retrieve it.  

Do you think we should update `getLabelText` to grab `component.ariaLabel` too?  The original issue pointed that out also, but like you said, that would result in multiple aria-labels, so probably not necessary and would be considered an anti-pattern?

Reply:
>  that would result in multiple aria-labels, so probably not necessary and would be considered an anti-pattern?

Yeah that's what I think.